orchestrator/api/knowledge_graph.py

```
@@ -92,34 +92,41 @@ async def list_entities(
         order_clause = order_map.get(sort_by, "importance_score DESC")
-        
+
         query_text = f"""
-            SELECT id, entity_name, entity_type, canonical_name, description, 
+            SELECT id, entity_name, entity_type, canonical_name, description,
                    mention_count, importance_score, created_at
             FROM kb_entities
             {where_clause}
             ORDER BY {order_clause}
             LIMIT :limit OFFSET :offset
         """
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)